### ISSUE TITLE
Several HTTP failures when running off an ingress vs. kubectl proxy

### ISSUE THREAD

**Issue by michaelajr** [kubecost/cost-analyzer-helm-chart#366]
There are several failures when running behind an ingress. When using `kubectl proxy` I do not get these errors.

Using an ingress:
![Screen Shot 2020-03-31 at 3 17 51 PM](https://user-images.githubusercontent.com/658602/78248450-a89fb780-74ba-11ea-9d37-92ce0e6063d4.jpeg)

Using kubectl proxy:
![Screen Shot 2020-04-02 at 8 15 18 AM](https://user-images.githubusercontent.com/658602/78248477-b3f2e300-74ba-11ea-995f-a2b3fa4b0f40.jpeg)

Here is the ingress YAML:
```
kind: Ingress
metadata:
  annotations:
    kubernetes.io/ingress.class: nginx
    nginx.ingress.kubernetes.io/configuration-snippet: |
      rewrite ^(/kubecost)$ $1/ permanent;
    nginx.ingress.kubernetes.io/enable-cors: "true"
    nginx.ingress.kubernetes.io/rewrite-target: /$2
  labels:
    app: cost-analyzer
    app.kubernetes.io/instance: kubecost
    app.kubernetes.io/managed-by: Helm
    app.kubernetes.io/name: cost-analyzer
    helm.sh/chart: cost-analyzer-1.55.1
  name: kubecost-cost-analyzer
  namespace: kube-system
spec:
  rules:
  - host: mycluster.mydomain.com
    http:
      paths:
      - backend:
          serviceName: kubecost-cost-analyzer
          servicePort: frontend
        path: /kubecost(/|$)(.*)
  tls:
  - hosts:
    - mycluster.mydomain.com
    secretName: my-ingress-tls
```

This ingress runs KubeCost on a path @ `/kubecost` but I have seen this running off the root as well. So I do not think it has anything to do with the path. I have always seen this when using an ingress.

**Comment by michaelajr** [comment]
I see this issue. The URLs are being constructed wrong. Will send email.

**Comment by dwbrown2** [comment]
Greatly appreciate the thorough report! This helped us a find a rewrite rule that was causing this issue based on your url. Fix now merged, expected in our release coming early next week.

I'll leave this open so we can update when prod release is available.

**Comment by dwbrown2** [comment]
@michaelajr this is addressed in v1.56.0 released today, let me know if we can get you anything else!

**Comment by michaelajr** [comment]
@dwbrown2 great will upgrade tonight/tomorrow. Thanks.
